contracts/interfaces/IFallbackManager.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+/**
+ * @title IFallbackManager - A contract interface managing fallback calls made to this contract.
+ * @author @safe-global/safe-protocol
+ */
+interface IFallbackManager {
```

Comment:
Hmm, I wonder if it makes sense for the events to also be part of the interface? @mmv08 and @akshay-ap - what do you think?

One argument for it, is that the generated ABIs from the Solidity compiler will include all the event information.

One argument against it is, is that the intention of introducing interfaces was for making on-chain calls to the Safe contracts easier, and adding events to the interfaces doesn't help at all there.

Reply:
Done 👍🏾